pkg/cmd/status/status.go

```diff
@@ -0,0 +1,616 @@
+			si.preview = e.Payload.Comment.Body
+			number = e.Payload.PullRequest.Number
+		case "IssueCommentEvent":
+			si.Reason = "comment on " + e.Payload.Issue.Title
+			si.preview = e.Payload.Comment.Body
+			number = e.Payload.Issue.Number
+		default:
+			continue
+		}
+		si.Repository = e.Repo.Name
+		si.Identifier = fmt.Sprintf("%s#%d", e.Repo.Name, number)
+		s.RepoActivity = append(s.RepoActivity, si)
+	}
+
+	return nil
+}
+
+func statusRun(opts *StatusOptions) error {
+	client, err := opts.HttpClient()
+	if err != nil {
+		return fmt.Errorf("could not create client: %w", err)
+	}
+
+	sg := NewStatusGetter(client, opts)
+	errc := make(chan error)
+
+	// TODO break out sections into individual subcommands
+
+	opts.IO.StartProgressIndicator()
+	go func() {
+		err := sg.LoadNotifications()
+		if err != nil {
+			err = fmt.Errorf("could not load notifications: %w", err)
+		}
+		errc <- err
+	}()
+
+	go func() {
+		err := sg.LoadEvents()
+		if err != nil {
+			err = fmt.Errorf("could not load events: %w", err)
+		}
+		errc <- err
+	}()
+
+	go func() {
+		err := sg.LoadSearchResults()
+		if err != nil {
+			err = fmt.Errorf("failed to search: %w", err)
+		}
+		errc <- err
+	}()
+
+	for i := 0; i < 3; i++ {
+		if err := <-errc; err != nil {
+			return err
+		}
+	}
+	opts.IO.StopProgressIndicator()
+
+	cs := opts.IO.ColorScheme()
+	out := opts.IO.Out
+	fullWidth := opts.IO.TerminalWidth()
+	halfWidth := (fullWidth / 2) - 2
+
+	idStyle := cs.Cyan
+	leftHalfStyle := lipgloss.NewStyle().Width(halfWidth).Padding(0).BorderRight(true).BorderStyle(lipgloss.NormalBorder())
+	rightHalfStyle := lipgloss.NewStyle().Width(halfWidth).Padding(0)
+
+	section := func(header string, items []StatusItem, width, rowLimit int) (string, error) {
+		tableOut := &bytes.Buffer{}
+		fmt.Fprintln(tableOut, cs.Bold(header))
+		tp := utils.NewTablePrinterWithOptions(opts.IO, utils.TablePrinterOptions{
+			IsTTY:    opts.IO.IsStdoutTTY(),
+			MaxWidth: width,
+			Out:      tableOut,
+		})
+		if len(items) == 0 {
+			tp.AddField("Nothing here ^_^", nil, nil)
+			tp.EndRow()
+		} else {
+			for i, si := range items {
+				if i == rowLimit {
+					break
+				}
+				tp.AddField(si.Identifier, nil, idStyle)
+				if si.Reason != "" {
+					tp.AddField(si.Reason, nil, nil)
+				}
+				tp.AddField(si.Preview(), nil, nil)
+				tp.EndRow()
+			}
+		}
+
+		err := tp.Render()
+		if err != nil {
+			return "", err
+		}
+
+		return tableOut.String(), nil
+	}
+
+	mSection, err := section("Mentions", sg.Mentions, halfWidth, 5)
+	if err != nil {
+		return fmt.Errorf("failed to render 'Mentions': %w", err)
+	}
+	mSection = rightHalfStyle.Render(mSection)
+
+	rrSection, err := section("Review Requests", sg.ReviewRequests, halfWidth, 5)
+	if err != nil {
+		return fmt.Errorf("failed to render 'Review Requests': %w", err)
+	}
+	rrSection = leftHalfStyle.Render(rrSection)
+
+	prSection, err := section("Assigned PRs", sg.AssignedPRs, halfWidth, 5)
```

Comment:
In UI, would we stay consistent with the rest of gh and try to use "pull request" instead of the abbreviation "PR" in UI? Wherever reasonable, of course, and I feel here is reasonable.